src/components/footer/_footer.scss

```diff
@@ -65,11 +70,7 @@
       }
     }
 
-    // When focussed, the text colour needs to be darker to ensure that colour
-    // contrast is still acceptable
-    &:focus {
-      color: $govuk-focus-text-colour;
-    }
+    @include govuk-focusable-text-link;
```

Comment:
Is the positioning of this important (coming after the pseudo-classes)? If so, might be worth a comment as I think we generally try and do our includes together at the top of the ruleset.

Reply:
Yeah we tend to put focus styles at the end of the block, I would prefer to have the mixin only contain properties to avoid this confusion.

It's on my list of things to consider once we've done all the work and know what the mixins need to do...